src/computability/partrec_code.lean

```diff
@@ -479,6 +512,17 @@ end
 
 end
 
+/--
+The inperpretation of a `nat.partrec.code` as a partial function.
```

Comment:
Perhaps this information should be associated to the constructors instead of the `eval` function?